package.json

```diff
@@ -23,9 +23,12 @@
   },
   "dependencies": {
     "bluebird": "^3.2.2",
-    "eslint": "^4.18.2",
     "firebase": "^2.3.2",
     "lodash.debounce": "^4.0.8",
-    "request-promise": "^1.0.2"
+    "request": "^2.88.0",
```

Comment:
just curious; why has "request" been added?

Reply:
The newer version of __request-promise__ designated that __request__ is a _peer dependency_ instead of a normal dependency, meaning that this project has to explicitly include __request__ as a dependency.

`npm install` has the following output if request is not included as a dependency:

```
npm WARN request-promise@4.2.4 requires a peer of request@^2.34 but none is installed. You must install peer dependencies yourself.
npm WARN request-promise-core@1.1.2 requires a peer of request@^2.34 but none is installed. You must install peer dependencies yourself.
```